doc/man3/PEM_read_bio_PrivateKey.pod

```diff
@@ -355,7 +355,8 @@ value as the I<u> parameter passed to the PEM routine. It allows
 arbitrary data to be passed to the callback by the application
 (for example a window handle in a GUI application). The callback
 I<must> return the number of characters in the passphrase or -1 if
-an error occurred.
+an error occurred. The passphrase can be arbitrary data, in case it is
```

Comment:
```suggestion
an error occurred. The passphrase can be arbitrary data; in the case where it is
```
nitty-nit: this is a comma splice, plus some other wording tweaks since I'm suggesting already.